src/assets/data/defaultConfig.ts

```diff
@@ -33,7 +33,7 @@ export const DefaultSettingsWeeklyReport: WeeklyReportSubSettings[] = [
 ]
 
 // DEFAULT SETUP
-export const DefaultTaskCompletionWindow = 259200000 // 3 days in ms
+export const DefaultTaskCompletionWindow = 86400000 // 1 day in ms
```

Comment:
Have you tried changing the completion window values in the protocols.json?

Reply:
I agree this needs to be tested with protocols too for 3 day window.

Reply:
Yes. That is the idea behind keeping it part of the protocol right.

Reply:
Yes this has been tested here with this protocol: https://github.com/RADAR-base/RADAR-aRMT-protocols/pull/23

Reply:
ok thanks